tf2_ros/src/buffer.cpp

```diff
@@ -44,10 +44,22 @@
 #define ROS_INFO printf
 #define ROS_WARN printf
 
-
 namespace tf2_ros
 {
 
+// Added to backport: https://github.com/ros2/geometry2/pull/281
+std::mutex g_object_map_to_cb_handle_mutex;
+std::map<void*, std::unordered_map<TimerHandle, tf2::TransformableCallbackHandle>> g_object_map_to_cb_handle;
```

Comment:
Nit: `#include <map>` and `#include <unordered_map>` for this.

Reply:
Also, we should mark both of these as either `static` or in an anonymous namespace just to keep the symbols private.

Reply:
And I believe we are always storing a `Buffer *` in here, so we can change the key of the map to `Buffer *`.